src/api/layers/KMLLayer.class.js

```diff
@@ -53,11 +57,7 @@ export default class KMLLayer extends AbstractLayer {
 
     getID() {
         // format coming from https://github.com/geoadmin/web-mapviewer/blob/develop/adr/2021_03_16_url_param_structure.md
-        let id = `KML|${encodeURIComponent(this.kmlFileUrl)}|${this.name}`
-        if (this.adminId) {
-            id += `@adminId=${this.adminId}`
-        }
-        return id
+        return `KML|${this.kmlFileUrl}|${this.name}`
```

Comment:
Just a short question for my comprehension. Is the admin ID not appearing anymore in the URL simply because you removed it from the ID here?